embedded-graphics/README.md

```diff
@@ -11,188 +11,250 @@
+[simulator]: https://github.com/jamwaffles/embedded-graphics/tree/c4f74c12dae9f0a0193fa48192f905a002bf8c9d/simulator
+[simulator examples]: https://github.com/jamwaffles/embedded-graphics/tree/c4f74c12dae9f0a0193fa48192f905a002bf8c9d/simulator/examples
 
-Note that some of these crates may not support the latest version of embedded-graphics.
+## Crate features
 
-If you know of a crate that is not in this list, please [open an
-issue](https://github.com/jamwaffles/embedded-graphics/issues/new).
+Add these to your `Cargo.toml` to turn on extra bits of functionality.
 
-## Display drivers
+* `nalgebra_support` - use the [Nalgebra](https://crates.io/crates/nalgebra) crate with `no_std`
+support to enable conversions from `nalgebra::Vector2` to `Point` and `Size` 
 
-- [embedded-graphics-simulator](https://docs.rs/embedded-graphics-simulator/): Simulated display
-- [embedded-graphics-web-simulator](https://crates.io/crates/embedded-graphics-web-simulator): Simulated display in your browser via Webassembly
-- [epd-waveshare](https://crates.io/crates/epd-waveshare) Driver for various ePaper displays (EPD) from Waveshare
-- [hub75](https://crates.io/crates/hub75): A rust driver for hub75 rgb matrix displays
-- [ili9341](https://crates.io/crates/ili9341): A platform agnostic driver to interface with the ILI9341 (and ILI9340C) TFT LCD display
-- [ls010b7dh01](https://crates.io/crates/ls010b7dh01): A platform agnostic driver for the LS010B7DH01 memory LCD display
-- [sh1106](https://crates.io/crates/sh1106): I2C driver for the SH1106 OLED display
-- [ssd1306](https://crates.io/crates/ssd1306): I2C and SPI (4 wire) driver for the SSD1306 OLED display
-- [ssd1322](https://crates.io/crates/ssd1322): Pure Rust driver for the SSD1322 OLED display chip
-- [ssd1331](https://crates.io/crates/ssd1331): SPI (4 wire) driver for the SSD1331 OLED display
-- [ssd1351](https://crates.io/crates/ssd1351): SSD1351 driver
-- [ssd1675](https://crates.io/crates/ssd1675): Rust driver for the Solomon Systech SSD1675 e-Paper display (EPD) controller
-- [st7735-lcd](https://crates.io/crates/st7735-lcd): Rust library for displays using the ST7735 driver
-- [st7789](https://crates.io/crates/st7789): A Rust driver library for ST7789 displays
-- [st7920](https://crates.io/crates/st7920): ST7920 LCD driver in Rust
+* `fixed_point` - use fixed point arithmetic instead of floating point for all trigonometric
+calculation.
 
-Note that some drivers may not support the latest version of embedded-graphics.
+## Migrating from 0.5 to 0.6
+
+Please read [the migration guide](https://github.com/jamwaffles/embedded-graphics/blob/master/embedded-graphics/MIGRATING-0.5-0.6.md).
 
-There may be other drivers out there we don't know about yet. If you know of a driver to add to this list, please open [an issue](https://github.com/jamwaffles/embedded-graphics/issues/new)!
+## Implementing `embedded_graphics` in a driver
```

Comment:
Is "Implementing `embedded_graphics`..." right in this context? Because you implement the driver API and not e-g itself.